src/http/fetcher.ts

```diff
@@ -22,51 +32,27 @@ export class Fetcher {
   protected _httpDriver: IHttpDriver;
   private _logger: Logger;
   private readonly _baseUrl: string;
-  private _clientState: IFetcherClientState;
+  private _clientState: IClientState;
   private readonly _baseApiURL: string;
 
-  constructor(logger: Logger, clientState: IFetcherClientState, baseUrl: string, httpDriver?: IHttpDriver) {
+  constructor(logger: Logger, clientState: IClientState, baseUrl: string, httpDriver?: IHttpDriver) {
     this._httpDriver = httpDriver ? httpDriver : new DefaultHttpDriver();
     this._logger = logger;
     this._clientState = clientState;
-    this._baseUrl = baseUrl.endsWith('/') ? baseUrl : (baseUrl + '/');
+    this._baseUrl = baseUrl.endsWith('/') ? baseUrl : baseUrl + '/';
```

Comment:
Be careful when merge to not accept this ```endsWith ``` over the previous fix

Reply:
endsWith was not the problem. The problem on IE was about the baseURI API. ButI will take care of it.